Numerics/EvaluatingFunctions/EvaluatePolynomial.qs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Numerics.Samples
+{
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Arithmetic;
+
```

Comment:
Similarly here, I'd suggest some introductory comments.

Reply:
I added a `README.md` that shortly introduces the 3 samples.